### PR TITLE
Fixes 5639, providing a migration, and moving 1856 to production

### LINKED ISSUE #5639: [1856] Extra Pass required even though no action is possible
During operation, when you take a loan early the game still requires you to pass last, even though you are not allowed to take more.
And that even if you do not have funds to repay any loan.

So the game should skip pass in these situations.

https://18xx.games/game/44377?action=133

image

### PR DESCRIPTION
**Pull request by ventusignis**
Fixes #5639

### CODE REVIEW

File: migrate_game.rb [hunk 1]
```diff
@@ -263,6 +274,19 @@ def repair(game, original_actions, actions, broken_action)
   elsif broken_action['type'] == 'dividend' and broken_action['entity_type'] == 'minor'
     actions.delete(broken_action)
     return
+  
+  elsif (broken_action['type'] != 'pass') &&
```
Comment (ventusignis) [inline migrate_game.rb:278]: Is this reasonable? The extra pass gets caught up in a few different ways but I'm wondering if this is too permissive

File: migrate_game.rb [hunk 2]
```diff
@@ -182,6 +182,16 @@ def repair(game, original_actions, actions, broken_action)
         return
       end
     end
+    if game.active_step.is_a?(Engine::Game::G1856::Step::BuySellParShares) ||
```
Comment (tobymao) [inline migrate_game.rb:185]: we're not gonna run this, and will just pin, so let's get rid of it

Comment (ventusignis) [reply migrate_game.rb:185]: That sounds good to me

File: assets/app/view/game_page.rb [hunk 3]
```diff
@@ -22,7 +22,7 @@ class GamePage < Snabberb::Component
     APP_PADDING_BOTTOM = '2vmin'
 
     def render_broken_game(e)
-      inner = [h(:div, "We're sorry this game cannot be continued due to #{e}")]
```
Comment (ventusignis) [inline assets/app/view/game_page.rb:25]: Removing this change

**Review by tobymao — APPROVED** [review]

**Pull request merged**